src/julia.h

```diff
@@ -1934,6 +1934,10 @@ typedef struct _jl_task_t {
     uint16_t priority;
 
 // hidden state:
+
+#ifdef USE_TRACY
+    const char *name;
+#endif
```

Comment:
Mentioned on the call, adding this note so it isn't lost: would be great to have a `name` irrespective of Tracy.